src/main/ipv-stub/ipv-authorize.ts

```diff
@@ -63,18 +64,134 @@ async function get(
     return errorResponse(500, "Decoding failed: " + message);
   }
 
+  const jsonBody = JSON.parse(decodedPayload);
+
   return {
     statusCode: 200,
     headers: { "Content-Type": "text/html" },
     body: renderPage(
-      `<h1 class="govuk-heading-l">Hello World from the IPV authorize lambda</h1>
+      `<h1 class="govuk-heading-l">IPV authorize lambda stub</h1>
   <h3 class="govuk-heading-s">Decrypted JAR header:</h3>
-  <p class="govuk-body">${JSON.parse(decodedHeader)}</p>
+  <dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+  <dt class="govuk-summary-list__key">
+  Algorithm
+  </dt>
+  <dd class="govuk-summary-list__value" id="user-info-core-identity-claim-present">
+  ${JSON.parse(decodedHeader).alg}
+  </dd>
+  </dl>
+
+
   <h3 class="govuk-heading-s">Decrypted JAR payload:</h3>
-  <p class="govuk-body">${JSON.parse(decodedPayload)}</p>
+  <dl class="govuk-summary-list">
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+        Core Identity Claim
+    </dt>
+    <dd class="govuk-summary-list__value" id="user-info-core-identity-claim">
+    ${JSON.stringify(jsonBody.claims.userinfo["https://vocab.account.gov.uk/v1/coreIdentityJWT"])} 
```

Comment:
It would be nice if the json on this page was formatted for readability, not super essential though